packages/terra-application-header-layout/src/ApplicationHeaderLayout.scss

```diff
@@ -0,0 +1,39 @@
+/* stylelint-disable selector-class-pattern */
+:local {
+  .fit {
+    flex: 0 0 auto;
+  }
+
+  .fill {
+    flex: 1 1 auto;
+    justify-content: space-between;
+    min-width: 0;
+    width: 100%;
+  }
+
+  .fit,
+  .fill { 
+    align-items: center;
+    display: flex;
+    height: 100%;
+    position: relative;
+  }
+
+  .start {
+    justify-content: flex-start;
+  }
+
+  .end,
+  .header-inner {
+    justify-content: flex-end;
+  }
+
+  .logo,
+  .utilities {
+    max-width: 40%;
```

Comment:
Instead of settings widths with percentage flowers, can we instead rely on the flex-grow properties https://css-tricks.com/snippets/css/a-guide-to-flexbox/#article-header-id-10?